src/shapes/input_text/InputText.js

```diff
@@ -14,9 +14,25 @@ const options = {
       type: 'dropdown',
       choices: ['Cursive', 'Sans Serif', 'Monospace'],
     },
+    rotateDir: {
+      title: 'Rotate',
+      type: 'dropdown',
+      choices: ['Top', 'Center', 'Bottom'],
```

Comment:
![image](https://user-images.githubusercontent.com/3259145/79167374-4ba6da00-7db5-11ea-819c-4a2d8bcfbc4f.png)

Do you think for bottom text, you should reverse the lines of text so that reading from top to bottom is the same?

Reply:
Great idea.